ledger/sandbox-common/src/main/scala/platform/sandbox/cli/CommonCliBase.scala

```diff
@@ -309,6 +300,11 @@ class CommonCliBase(name: LedgerName) {
           s"By default compression is off, this switch enables it. This has only effect for append-only ingestion." // TODO append-only: fix description
         )
 
+      com.daml.cliopts.Metrics.metricsReporterParse(this)(
+        (f, c) => c.copy(metricsReporter = f(c.metricsReporter)),
```

Comment:
To be consistent with the style already established, please don't use abbreviations:
```suggestion
        (setter, config) => config.copy(metricsReporter = setter(config.metricsReporter)),
```